base/util_testing.go

```diff
@@ -130,7 +130,9 @@ func getTestBucket(t testing.TB, persistent bool) *TestBucket {
 func (tb *TestBucket) GetNamedDataStore(count int) (DataStore, error) {
 	dataStoreNames := tb.GetNonDefaultDatastoreNames()
 	if count > len(dataStoreNames) {
-		tb.t.Errorf("You are requesting more datastores %d than are available on this test instance %d", dataStoreNames, count)
+		tb.t.Fatalf("You are requesting more datastores %d than are available on this test instance %d", dataStoreNames, count)
```

Comment:
Parameters are wrong way around (and missing `len()` for `%d`)

```suggestion
		tb.t.Fatalf("You are requesting more datastores %d than are available on this test instance %d", count, len(dataStoreNames))
```